build_runner/test/generate/build_test.dart

```diff
@@ -59,7 +60,8 @@ void main() {
         await testBuilders([
           apply('', '', [(_) => new CopyBuilder(extension: '1')], toRoot(),
```

Comment:
should this have a real builder key as well? do we do any validation to make sure we don't have duplicate builder keys, or does it not matter?

Reply:
We don't need to match up with any config so we shouldn't need it. Duplicate keys don't matter for now.